docs/index.md

```diff
@@ -55,8 +55,8 @@ Streamlit works, see [Main concepts](main_concepts.md).
 
 Follow these steps and you'll have a sample app running in less than 5 minutes.
 
-1. Before you do anything else, make sure that you have [Python 2.7.0 or later
-   / Python 3.6.x or later](https://www.python.org/downloads/).
+1. Make sure that you have [Python 2.7.0
+   / Python 3.6.x or later](https://www.python.org/downloads/) installed.
```

Comment:
We should not remove the "or later" next to Python 2.7.0, as alternative it could say "Make sure that you have [Python 2.7.x / 3.6.x](https://www.python.org/downloads/) installed."

Reply:
Does that capture the fact that folks can use 3.7/3.8? just making sure we aren't making the readers think they're limited to 3.6.x. I'll change it for now, but something we should think about. What do you think @tvst?

Reply:
Just keep it with the 'or later':

Make sure that you have [Python 2.7.x  / Python 3.6.x or later]